internal/pkg/service/buffer/worker/distribution/node.go

```diff
@@ -211,11 +209,6 @@ func (n *Node) watch(ctx context.Context, wg *sync.WaitGroup) error {
 		}
 	}()
 
-	// Wait for self-discovery
-	if err := <-selfDiscovery; err != nil {
-		return err
-	}
-
```

Comment:
Pri vytvarani `distribution.Node` sa cakalo na self-discovery, ... kym sa node dozvie sama o sebe, spatne cez etcd.

Lenze to vlastne nepotrebujeme, kedze sa fixol ten problem s `initDone` (vid riadok nizsie `<-initDone`). To nam zabezpecuje nacitanie vsetky nodes.

A este je tam pridany na riadku `217` self-discovery test, ak by tam bol nejaky bug.

TLDR: uz sa cakat nemusi, je to jednoduchsie.